lib/internal/http2/compat.js

```diff
@@ -46,6 +46,7 @@ const {
 } = constants;
 
 let statusMessageWarned = false;
+const statusHeaderMessageWarned = {};
```

Comment:
We shouldn't make this be a dictionary until we actually know we will need that. For now, just have it be `connectionHeaderWarned = false;`